### PR TITLE
Adding additional functionality to triangulation function

### PR DESCRIPTION
**Pull request by sean-afshar**
adding:
- ability to select frame ranges 
- ability to exclude certain views

### CODE REVIEW

File: sleap_anipose/calibration.py [hunk 1]
```diff
@@ -222,29 +230,24 @@ def get_metadata(
...
     """Generate movies from calibration board images.
 
     Args:
-        session: Path pointing to the session with the calibration board images.
+        view: Path pointing to the view subfolder with the calibration board images.
     """
-    cams = [x for x in Path(session).iterdir() if x.is_dir()]
-
-    for cam in cams:
+    # cams = [x for x in Path(session).iterdir() if x.is_dir()]
```
Comment (talmo) [inline sleap_anipose/calibration.py]: Remove

File: tests/test_calibration.py [hunk 2]
```diff
@@ -8,12 +8,19 @@
 import toml
 
 
-def test_calibrate(minimal_session, tmp_path):
+def test_calibrate(minimal_session, tmp_path, excluded_views=("side")):
```
Comment (talmo) [inline tests/test_calibration.py]: Don't set this in the input arguments to this test function. If you want to run the test with different excluded views, use pytest's `@parametrize` decorator.

Also, `("side")` will be the same as `"side"`. To specify a 1-element tuple, you have to use the syntax: `("side",)` -- note the dangling comma.

File: tests/test_triangulation.py [hunk 3]
```diff
@@ -44,18 +57,24 @@ def test_reproject(minimal_session):
...
+        for x in Path(minimal_session).iterdir()
+        if x.is_dir() and x.name not in excluded_views
+    ]
     assert p2d.shape[0] == len(cams)
+    assert p2d.shape[1] == frames[1] - frames[0]
     assert p2d.shape[-1] == 2
 
 
-def test_load_view(minimal_session):
+def test_load_view(minimal_session, frames=(25, 75)):
```
Comment (talmo) [inline tests/test_triangulation.py]: Same as above -- arguments to testing functions should not be used to specify constants.

**Review by talmo — CHANGES_REQUESTED** [review]

File: sleap_anipose/calibration.py (hunk 1 above)
Comment (sean-afshar) [reply sleap_anipose/calibration.py]: resolved

File: tests/test_triangulation.py (hunk 3 above)
Comment (sean-afshar) [reply tests/test_triangulation.py]: resolved

File: tests/test_calibration.py (hunk 2 above)
Comment (sean-afshar) [reply tests/test_calibration.py]: resolved

**Review by talmo — APPROVED** [review]

**Pull request merged**